mem-27-extra.xml

```diff
@@ -17532,7 +17532,7 @@ Erityiset kiitokset Marc Okrandille ({marq 'oqranD:n:name}) klingonin kielen luo
       <column name="definition_ru">поле? (физическая концепция) [AUTOTRANSLATED]</column>
       <column name="definition_zh_HK">領域？ （物理概念） [AUTOTRANSLATED]</column>
       <column name="definition_pt">campo? (conceito de física)</column>
-      <column name="definition_fi">kenttä? (fysiikan käsite)</column>
+      <column name="definition_fi">(kenttä? (fysiikka))</column>
```

Comment:
Do you want to add a comment in the README.md in the Finnish section about this convention?